editor/js/libs/app.js

```diff
@@ -180,6 +180,7 @@ var APP = {
 
 			prevTime = performance.now();
 
+			if ( renderer === undefined ) return;
```

Comment:
What case does this solve?

Reply:
That solves the case when I try to stop the execution If it does not exist.
If `signals.stopPlayer.dispatch()` is triggered but there's no script running `app.js` will fail because renderer is undefined. I could not found any "isPlaying() method" to avoid placing undefined checks elsewhere.